solr/solr-ref-guide/src/format-of-solr-xml.adoc

```diff
@@ -103,6 +103,9 @@ Specifies the path to a common library directory that will be shared across all
 `allowPaths`::
 Solr will normally only access folders relative to `$SOLR_HOME`, `$SOLR_DATA_HOME` or `coreRootDir`. If you need to e.g., create a core outside of these paths, you can explicitly allow the path with `allowPaths`. It is a comma separated string of file system paths to allow. The special value of `*` will allow any path on the system.
 
+`allowUrls``:
+When running Solr in non-cloud mode and if planning to do distributed search (using the "shards" parameter), the list of hosts needs to be allowed or Solr will forbid the request. The allow-list can also be configured in `solr.in.sh`.
```

Comment:
Instead of making reference so `solr.in.sh`, it's better to refer to the particular environment variable that is used.  People can set that env variable even if they don't touch solr.in.sh.

Reply:
And since there are more extensive docs on the `distributed-requests.adoc` page; lets link to there.  Or move more of that info here as you prefer.